src/TorchSharp/NN/Module.cs

```diff
@@ -415,7 +452,9 @@ public virtual string GetName()
                     return res;
                 }
 
-                public virtual Tensor forward(Tensor t) => throw new NotImplementedException("forward");
+                public virtual Tensor forward(Tensor t) => throw new NotImplementedException("forward(t)");
+
+                public virtual Tensor forward(Tensor x, Tensor y) => throw new NotImplementedException("forward(x,y)");
```

Comment:
It might be confusing, that overriding this function from C# on a custom module has no effect.

Reply:
@lostmsu -- please elaborate. I'm not sure why it won't have any effect, I'm missing something.

Reply:
I was referring to the 2 parameter overload. It is not used when the `THSNN_custom_module` is called.

Reply:
Yes, that is a complex problem -- the fact that the Pytorch forward() really takes a variable-length argument list, so currently Sequential will only forward to the single-argument forward(), you have to build your own custom forward() to do anything else, like a couple of the examples do. Unfortunately, it also interacts with default arguments declared in Python, etc., so it's far from straight-forward to fix.

I'll punt on that in this PR, and open a specific issue to address the problem.